src/extension/flutter/flutter_daemon.ts

```diff
@@ -65,6 +71,16 @@ export class FlutterDaemon extends StdIOService<UnknownNotification> implements
 		}
 	}
 
+	protected createNcProcess(port: string) {
```

Comment:
It may be worth some comments on this method explaining roughly what the setup is here.

I _think_ the setup is that the user already has a tunnel set up from the remote machine to the local machine, and this is causing the extension (running in the remote machine) to connect over that tunnel to a deamon on the local machine instead of spawning its own (so that it can discover the local devices)?